packages/gafl-webapp-service/src/routes/journey-definition.js

```diff
@@ -66,7 +68,10 @@ export default [
         page: LICENCE_TO_START
       }
     },
-    backLink: s => (s.fromSummary ? LICENCE_SUMMARY.uri : null)
+    backLink: async status => {
+      const backLinkResult = await backLinkHandler(status, null)
+      return backLinkResult
+    }
```

Comment:
this would be neater as `backLink: status => backLinkHandler(status, null)`. I also think that `backLink: backLinkHandler` would work too.